src/main/kotlin/org/ionproject/integration/domain/evaluations/BusinessObjects.kt

```diff
@@ -146,7 +146,7 @@ data class Evaluations(
                             )
                         examList.add(
                             Exam(
-                                cleanedLine[TableColumnWinterCourse.COURSE.ordinal].text,
+                                trimCourse(cleanedLine[TableColumn.COURSE.ordinal].text),
```

Comment:
since we're repeating this call on each add maybe it should just be called once and saved on a variable?

Reply:
Removed from the Exam instance creation and added to the beginning of the line iteration